openlibrary/components/BarcodeScanner/components/LazyBookCard.vue

```diff
@@ -0,0 +1,142 @@
+<template>
+  <a class="lazy-book-card" :href="link">
+    <div class="cover">
+      <img :src="coverSrc">
+    </div>
+    <div class="info">
+      <div class="title">{{ title }}</div>
+      <div class="byline">{{ byline }}</div>
+      <div class="identifier">{{ identifier }}</div>
+    </div>
+  </a>
+</template>
+
+<script>
+import CONFIGS from '../../configs';
+import Promise from 'promise-polyfill';
+export default {
+    props: {
+        isbn: {
+            type: String || null,
+        },
+        tentativeCover: {
+            type: String || null,
+        },
```

Comment:
You can't do complex types like that in Vue, but this should do the trick! Also ISBN is not optional, so it actually can't be null!
```suggestion
        isbn: {
            type: String,
        },
        tentativeCover: {
            type: String,
            default: null,
        },
```